iroh-net/src/magicsock/endpoint/best_addr.rs

```diff
@@ -0,0 +1,202 @@
+//! The [`BestAddr`] is the currently active best address for UDP sends.
+
+use std::{
+    net::SocketAddr,
+    time::{Duration, Instant},
+};
+
+use iroh_metrics::inc;
+use tracing::{debug, info, trace};
+
+use super::{AddrLatency, TRUST_UDP_ADDR_DURATION};
```

Comment:
These two only exist for the best address, maybe move them to this file? Having a separate file is a really good call, helping cleanup `endpoint.rs` and my moving we can further tidy it